components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/WriterGroupMessageSource.cs

```diff
@@ -30,6 +30,18 @@ public class WriterGroupMessageTrigger : IMessageTrigger, IDisposable {
         public int NumberOfConnectionRetries => _subscriptions?.FirstOrDefault()?.
             Subscription?.NumberOfConnectionRetries ?? 0;
 
+        /// <inheritdoc/>
+        public bool IsConnectionOk => _subscriptions?.FirstOrDefault()?.
+            Subscription?.IsConnectionOk ?? false;
+
+        /// <inheritdoc/>
+        public int NumberOfGoodNodes => _subscriptions?.FirstOrDefault()?.
```

Comment:
@peterbryntesson, why are we getting `NumberOfGoodNodes` only from the first subscription and not summing over all of them ? I have the same question about all new counters in this class, as well as `NumberOfConnectionRetries` which I see was already present. Can we please test that if `WriterGroupMessageTrigger` has several subscriptions in it then the reported values would still be valid ?

Reply:
I just followed the pattern with NumberOfConnectionRetries. Let me change that. I will change it for NumberOfConnectionRetries as well.

Reply:
Done